README.md

```diff
@@ -22,12 +22,38 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
 
 ## Building
 
--   Make sure `ffmpeg` is on your path
--   Install Node.js and Yarn
--   Install Java (required for textures)
--   Run `yarn` in the root folder
--   Cd into `gulp` folder
--   Run `yarn` and then `yarn gulp` - it should now open in your browser
+Note that this process is for Windows. Use equivalent actions for other operating systems. This guide is also meant to be understandable by users with little to no knowledge of Git.
+
+(credit to Hyperion-21)
+
+**SETUP**
```

Comment:
These instructions are still for Windows only

Reply:
> Note that this process is for Windows. Use equivalent actions for other operating systems.

> also it might be helpful to add that for other systems, setup is whatever equivalent they have, but I don't know much about other systems

?

Reply:
shh